### PR TITLE
Mention missing null propagation in streams

### PR DESCRIPTION
**Pull request by vitusortner**
#338

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -454,12 +454,11 @@ Future<int> deletePersons(List<Person> person);
 ```
 
 ### Streams
-As already mentioned, queries can not only return a value once when called but also a continuous stream of query results.
-The returned stream keeps you in sync with the changes happening to the database table.
-This feature plays really well with the `StreamBuilder` widget.
+As already mentioned, queries cannot only return values once when called but also continuous streams of query results.
```
Comment (mqus) [inline README.md:457]: This "cannot" seems weird here... I think it was better before.
But I'm not a native, so I don't really know.

Comment (vitusortner) [reply README.md:457]: My IDE suggested that change. Apparently "cannot" is used more frequently compared to "can not". https://www.grammarly.com/blog/cannot-or-can-not/

Comment (mqus) [reply README.md:457]: I would usually agree, but the combination with "only" is what triggered me... 
well, it's not that important :smile:

Comment (mqus) [reply README.md:457]: *evil laugh* found something confirming my position: https://english.stackexchange.com/questions/421944/can-we-use-the-wording-we-can-not-only-but-also

Comment (vitusortner) [reply README.md:457]: Haha I see 😄

**Pull request merged**